### DISCUSSION TITLE
Instagram icon

### DISCUSSION THREAD

**Discussion by godcanonen** [Godiesc/firefox-gx#193] [Q&A]
I added Instagram to my sidebar in firefox, and decided to change its icon to original but couldn't find its svg icon location. I looked for it in chrome/icons but still didn't find

**Comment by Godiesc** [comment]
hi,
that icon extension the theme don't change at all, the icon you see is the icon of the extension, if you want to change to the icon you want you can do it with CSS, add something like this in `ogx_tricks.css` file:

```
#sidebarinstachat_godie-BAP{
        list-style-image: url(../icons/logo_instagram.svg) !important;
    }
```

the icon you should put in 'icons' folder.
The default icon of that extension isn't the original for instagram for copyright right and if you use it Mozilla disable the extension.
